createdisk.sh

```diff
@@ -146,7 +146,11 @@ ssh -o StrictHostKeyChecking=no -i id_rsa_crc core@api.crc.testing -- sudo podma
 
 # Shutdown the instance
 sudo virsh shutdown ${VM_PREFIX}-master-0
-
+# Wait till instance shutdown gracefully
+until sudo virsh domstate ${VM_PREFIX}-master-0 | grep shut; do
```

Comment:
Initially I wanted to suggest using `virsh event` rather than doing this polling, but on second thought this would be racy as the VM could stop before `virsh event` starts. `virsh domstate` can be localized though (on my system `shut off` is `fermé`) so you'll need to set either `LANG` or `LC_ALL`.